array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,13 @@
+/* exported compact */
+function compact(array) {
+  const newArray = [];
+
+  for (let i = 0; i < array.length; i++) {
+    if (!!array[i] === true) {
+      newArray.push(array[i]);
+    } else {
+      continue;
+    }
+  }
+  return newArray;
+}
```

Comment:
Here's another solution on how we can accomplish this! We essentially just want to check if each value in the array is `truthy` or `falsy`. Since a conditional statement will always evaluate to true or false, we literally just need to check each item in the array.
```
function compact(array) {
  var output = [];
  for (var i = 0; i < array.length; i++) {
    var element = array[i];
    if (element) output.push(element);
  }
  return output;
}
```